File: maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.py
class Solution:
    def mostWordsFound(self, sentences: List[str]) -> int:
        array = []
        for sentence in sentences:
            count = 0
            words = sentence.split(' ')
            for word in words:
                count += 1
            array.append(count)
        print(array)
        max_val = max(array)
        print(max_val)
        return max_val
            
            
            
        

File: maximum-number-of-words-found-in-sentences/maximum-number-of-words-found-in-sentences.java
class Solution {
    public int mostWordsFound(String[] sentences) {
        int max_val = 0;
        for(int i=0; i<sentences.length; i++){
             max_val = Math.max(max_val, sentences[i].split(" ").length);
        }
        return max_val;   
    }
}